app/javascript/components/components/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Card, Button, CardFooter, CardBody,
+    CardTitle, CardText } from 'reactstrap';
+  
+  const Footer = (props) => {
+    return (
+      <div>
+        <Card>
+          <CardBody>
+            <CardTitle tag="h5">Footer Title</CardTitle>
+            <CardText>Created by Men In Black.</CardText>
+            <Button>Button</Button>
+          </CardBody>
+          <CardFooter className="text-muted">Footer</CardFooter>
+        </Card>
```

Comment:
perhaps reconsider different component or simplifying the markup to not use Card component here. Using a card in the footer seems like it's doing too much for the footer